### PR TITLE
Vocabulary - Khaled

### PR DESCRIPTION
**Pull request by Levaii**

### CODE REVIEW

**Review by gelilaa — DISMISSED** [review]
@Levaii can you please fix the conflict?

**Comment by arnochauveau** [comment]
@Levaii Can you please revert the changes other than `vocabulary/workflow.md` ? 
These seem to be accidental changes and also the source of your merge conflicts.

You can do so by running the following commands in your terminal: 

```bash
git checkout origin main -- README.md
git checkout origin main -- student-bios/Tahmina.md
```

You can then commit the changes and push them.

**Comment by Levaii** [comment]
hey @arnochauveau
I did revert the changes and did those commands, it seems to be the same issue, any tip?

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 > - [Discussions](https://github.com/lab-antwerp-1/home/discussions/)
 > - [Deliverables](https://github.com/lab-antwerp-1/home/projects/1)
 > - [Random Groups](https://lab-antwerp-1.github.io/home/randomizer)
+>
```
Comment (arnochauveau) [inline README.md:11]: Please revert changes to this file. You can use `git checkout origin main -- README.md`

File: student-bios/Tahmina.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # tahminarasoli
+
```
Comment (arnochauveau) [inline student-bios/Tahmina.md:2]: please revert changes to this file, you can use `git checkout origin main -- student-bios/Tahmina.md`

**Review by arnochauveau — CHANGES_REQUESTED** [review]

**Comment by arnochauveau** [comment]
> hey @arnochauveau I did revert the changes and did those commands, it seems to be the same issue, any tip?

Did you add,  commit and push after? 
```bash
git add .
git commit -m "revert whitespace changes"
git push
```

**Comment by Levaii** [comment]
@arnochauveau Yes I added a commit and pushed it.

**Comment by arnochauveau** [comment]
> @arnochauveau Yes I added a commit and pushed it.

I'm sorry, the right command was `git checkout origin/main -- README.md`, my bad. I've fixed it for you! 🙂

**Review by arnochauveau — APPROVED** [review]

**Pull request merged**